swift/apple/FirezoneNetworkExtension/ConfigurationManager.swift

```diff
@@ -69,6 +79,10 @@ class ConfigurationManager {
     setFirezoneId(UUID().uuidString)
   }
 
+  @objc private func handleUserDefaultsChanged(_ notification: Notification) {
+    self.managedDict = userDefaults.dictionary(forKey: managedDictKey) ?? [:]
```

Comment:
So we don't need to grab a new instance of `UserDefaults` but the same pointer is still valid? That is interesting. I guess it just re-reads them from the disk on access.

Reply:
I think stuff like this would be good to write a debug / info log for. I'd recommend to do another pass over all this config code and add a few logs to the happy path. Might be useful to see what values got changed, when etc.

Reply:
Yeah it's a single shared instance, `UserDefaults.standard`.

Hm yeah I'll look into it. We actually do get some logs from `cfprefsd` the preference daemon which powers this under the hood, but it might be good to have something on the app side, perhaps...